proxy/http/HttpConfig.h

```diff
@@ -358,6 +361,37 @@ struct HttpConfigPortRange {
   }
 };
 
+namespace HttpForwarded
+{
+// Options for what will be included in "Forwarded" field header.  Not mutually exclusive, except that there should be only one
+// "by" option enabled.  (I did not use enum class because the enum values would not implicitly convert to integral
+// types).
+//
+namespace Option
+{
+  enum E {
+    For,
+    ByIp,         // by=<numeric IP address>.
+    ByUnknown,    // by=unknown.
+    ByServerName, // by=<configured server name>.
+    ByUuid,       // Obfuscated value for by, by=_<UUID>.
+    Proto,        // Basic protocol (http, https) of incominng message.
+    Host,         // Host from URL before any remapping.
+    Connection,   // Verbose protocol from Via: field, with dashes instead of spaces.
```

Comment:
I thought we would have 3 options for `Connection` - compact, standard, full.

Reply:
Compact would be the same as "proto", you want it anyway?

Reply:
Yes, just for consistency with other uses.